set-versions

```diff
@@ -38,7 +38,7 @@ for project in $(find . -name project.clj -exec dirname '{}' \;); do
 
   cljproj=$(basename $project)
 
-  if [ "$cljproj" == "Donkey" ]; then
+  if [ "$cljproj" == "Terrain" ]; then
```

Comment:
Actually, we already have an automated script to update our services versions at the top level of the repo: [set-versions](https://github.com/iPlantCollaborativeOpenSource/DE/blob/master/set-versions).
It would be nice if it could also update the UI version in the ansible properties template too, but I'm not sure if that would be easy to accomplish.